Gemfile

```diff
@@ -3,16 +3,19 @@ source 'https://rubygems.org'
 # Going forward, these should be updated to the latest versions immediately post release
 chefspec_version = '= 7.0.0'
 foodcritic_version = '= 11.0.0'
+kitchen_inspec_version = '= 0.24.0'
+kitchen_vagrant_version = '1.3.4'
 rubocop_version = '= 0.48.1'
 chef_vault_version = '> 3.0'
 
-# https://github.com/cerner/cerner_splunk/issues/142
-chef_version = '= 12.18.31'
+chef_version = '= 14.5.27'
 
 gem 'berkshelf'
 gem 'chef', chef_version
 gem 'chef-sugar'
 gem 'chef-vault', chef_vault_version
 gem 'chefspec', chefspec_version
 gem 'foodcritic', foodcritic_version
+gem 'kitchen-inspec', kitchen_inspec_version
+gem 'kitchen-vagrant', kitchen_vagrant_version
```

Comment:
minor, why add these?  test kitchen isn't used in this cookbook (no .kitchen.yml file, no test/integration directory).

Reply:
I removed this in the new commit, sorry I was prepping to use test-kitchen for converging before finding another solution.